README.md

```diff
@@ -1,26 +1,80 @@
 # Ember-impagination
 [![Build Status](https://travis-ci.org/thefrontside/ember-impagination.svg)](https://travis-ci.org/thefrontside/ember-impagination)
 
-This README outlines the details of collaborating on this Ember addon.
+[Impagination](https://github.com/flexyford/impagination) implementation for Ember.
+
+## Demo
+[Ember-Impagination Demo](https://github.com/thefrontside/ember-impagination/tree/color-demo)
+The color-demo presents a finite scroll implementation of ember-impagination. Starting at RED, we scroll through all of the ROYGBIV color spectrum, fetching pages of colors dependent upon passed-in attributes. The color-demo contains implementations for [{{virtual-each}}](https://github.com/jasonmit/virtual-each) and [{{ember-collection}}](https://github.com/emberjs/ember-collection). *virtual-each* is the recommend ember-scroll-component for *ember-impagination*.
 
 ## Installation
 
-* `git clone` this repository
-* `npm install`
-* `bower install`
+* `ember install ember-impagination`
 
-## Running
+## Usage
```

Comment:
In the `Usage` section, I think most people will be using the component, so let's optimize for that case first. Afterwards, you can say "under the covers, the imagination-dataset component is just creating an impagination dataset. You can use it directly if you want. Here's how.